diffs/devguide.diff

```diff
@@ -0,0 +1,688 @@
+html/en/beats/devguide/master/_migrating_dashboards_from_kibana_5_x_to_6_x.html
+html/en/beats/devguide/master/archive-dashboards.html
+html/en/beats/devguide/master/beater-interface.html
+--- /dev/fd/63

```

Comment:
leaving tabs in place because tabs are preferred for golang (better copy/paste experience for users)

Reply:
so no changes required for this diff....

Reply:
++. This was a nice part of the change to asciidoctor.